toolchain/check/handle_binding_pattern.cpp

```diff
@@ -99,16 +99,28 @@ static auto HandleAnyBindingPattern(Context& context, Parse::NodeId node_id,
 
       // A `var` declaration at class scope introduces a field.
       auto parent_class_decl = context.GetCurrentScopeAs<SemIR::ClassDecl>();
-      cast_type_id = context.AsCompleteType(cast_type_id, [&] {
-        CARBON_DIAGNOSTIC(IncompleteTypeInVarDecl, Error,
-                          "{0} has incomplete type `{1}`", llvm::StringLiteral,
-                          SemIR::TypeId);
-        return context.emitter().Build(type_node, IncompleteTypeInVarDecl,
-                                       parent_class_decl
-                                           ? llvm::StringLiteral("Field")
-                                           : llvm::StringLiteral("Variable"),
-                                       cast_type_id);
-      });
+      cast_type_id = context.AsCompleteType(
+          cast_type_id,
+          [&] {
+            CARBON_DIAGNOSTIC(IncompleteTypeInVarDecl, Error,
+                              "{0} has incomplete type `{1}`",
+                              llvm::StringLiteral, SemIR::TypeId);
+            return context.emitter().Build(
+                type_node, IncompleteTypeInVarDecl,
+                parent_class_decl ? llvm::StringLiteral("Field")
+                                  : llvm::StringLiteral("Variable"),
+                cast_type_id);
+          },
+          [&] {
+            CARBON_DIAGNOSTIC(AbstractTypeInVarDecl, Error,
+                              "{0} has abstract type `{1}`",
+                              llvm::StringLiteral, SemIR::TypeId);
+            return context.emitter().Build(
+                type_node, AbstractTypeInVarDecl,
+                parent_class_decl ? llvm::StringLiteral("Field")
+                                  : llvm::StringLiteral("Variable"),
```

Comment:
```suggestion
                parent_class_decl ? llvm::StringLiteral("field")
                                  : llvm::StringLiteral("variable"),
                cast_type_id);
          },
          [&] {
            CARBON_DIAGNOSTIC(AbstractTypeInVarDecl, Error,
                              "{0} has abstract type `{1}`",
                              llvm::StringLiteral, SemIR::TypeId);
            return context.emitter().Build(
                type_node, AbstractTypeInVarDecl,
                parent_class_decl ? llvm::StringLiteral("field")
                                  : llvm::StringLiteral("variable"),
```
Let's fix the case of these while we're here.